<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Home</class>
 <widget class="QWidget" name="Home">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>559</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>149</y>
     <width>581</width>
     <height>221</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>579</width>
      <height>219</height>
     </rect>
    </property>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>561</width>
       <height>151</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QTextBrowser" name="textBrowserOfPost">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>361</width>
        <height>131</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="Number_of_like">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>80</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>number of like</string>
      </property>
     </widget>
     <widget class="QLabel" name="numberComment">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>80</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>number of comment</string>
      </property>
     </widget>
     <widget class="QPushButton" name="CommentButton">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>110</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Comment</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ShareButton">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>110</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>share</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="LikeButton">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>60</y>
        <width>70</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>like</string>
      </property>
     </widget>
     <widget class="QPushButton" name="SeeProfile">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>20</y>
        <width>75</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>see profile</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>10</y>
        <width>47</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>from :</string>
      </property>
     </widget>
     <widget class="QLabel" name="UserName">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>30</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>User name </string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="profile_button">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>111</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>Profile</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>109</y>
     <width>571</width>
     <height>43</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="homeButton">
      <property name="text">
       <string>home</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="NetworkButton">
      <property name="text">
       <string>Network</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="NewPostButton">
      <property name="text">
       <string>New Post</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="NotificationButton">
      <property name="text">
       <string>Notification</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="JobsButton">
      <property name="text">
       <string>Jobs</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="search_edit">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>50</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>30</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>search :</string>
   </property>
  </widget>
  <widget class="QPushButton" name="message_button">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>80</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>message</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
